docs/content.zh/docs/ops/state/disaggregated_state.md

```diff
@@ -150,6 +150,18 @@ state.backend.forst.primary-dir: s3://your-bucket/forst-state
 checkpoint and fast recovery, since the ForSt will perform file copy between the primary
 storage location and the checkpoint directory during checkpointing and recovery.
 
+#### ForSt Local Storage Location
+
+ForSt stores the state of synchronous operators in the local disk by default, to avoid the high
```

Comment:
```suggestion
By default, ForSt will **ONLY** disaggregate state when asynchronous APIs (State V2) are used. When using synchronous state APIs in Datastream and SQL jobs, ForSt will only serve as **local state store**. Since a job
may contain multiple ForSt instances with API mixed usage, synchronous local state access along with asynchronous remote state access could help achieve better overall throughput. If you ....... 
```